packages/components/popover/src/Popover.doc.mdx

```diff
@@ -43,6 +44,8 @@ A basic popover should have a trigger element and a content element.
 The trigger is the element that should toggle the popover.
 The content is the popover itself.
 
+**Note: For headings inside Popover, please use the `header` HTML tag instead of `h1, h2`, etc.**
```

Comment:
I discussed having a `Popover.Header` and `Popover.Footer` with Mathieu to explain to him why I think it's unnecessary (if we add one, it should be visible on the "Anatomy" section of the ZeroHeight documentation too)

If it is part of the anatomy specs should reflect it :) 

Basically it would look like this:
```
<Popover>
        <Popover.Trigger asChild>
          <Button>Trigger popover</Button>
        </Popover.Trigger>
        <Popover.Portal>
          <Popover.Content>
            <Popover.Header>Title</Popover.Header>
            <p>Are you sure you want to have that cookie now ?</p>
            <Popover.Arrow />
            <Popover.Close aria-label="Close the popover" />
          </Popover.Content>
        </Popover.Portal>
      </Popover>
```

Then should `Popover.Content` be a flex-col with a gap between elements then ? Or should the `Popover.Header` have a margin below by default ?

Reply:
Ok then we should also update the docs :+1: